Misc/NEWS.d/next/Library/2024-02-11-07-31-43.gh-issue-82062.eeS6w7.rst

```diff
@@ -0,0 +1,2 @@
+Fix :func:`inspect.signature()` to correctly set extension module for
+built-in instance methods.
```

Comment:
This description does not look correct. What module? The Signature object does not have any module attribute.

Please document what is actual visible effect of this change.